### PR TITLE
ENG-430 Rework Benthos UMH page

### PR DESCRIPTION
**Pull request by ricci2511**
# Description

The Benthos UMH feature page is updated to reflect the new/upcoming "Universal Protocol Converter" feature of the Management Console, allowing to deploy any supported protocol by Benthos UMH directly from the web app (not only OPC-UA anymore).

## Related issues

- Fixes ENG-430

## Checklist

*You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [contributing guidelines](https://umh.docs.umh.app/docs/development/contribute/new-content/add-documentation/) and the [code of conduct](https://github.com/united-manufacturing-hub/umh.docs.umh.app/blob/main/.github/CODE_OF_CONDUCT.md).
- [x] I have followed the [documentation](https://umh.docs.umh.app/docs/development/contribute/documentation/style/) style.

## Additional information

Before merging this we should probably ensure that everything related to the Universal Protocol feature is merged into prod.

### CODE REVIEW

File: umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md [hunk 1]
```diff
@@ -2,8 +2,7 @@
 title: Benthos UMH
 menuTitle: Benthos UMH
 description: |
-  Configure OPC-UA data sources to stream data to Kafka directly in the
-  Management Console.
+  Configure protocol converters to stream data to Kafka directly in the Management Console.
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md:5]: stream data to the Unified Namespace

File: umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md [hunk 2]
```diff
@@ -22,10 +21,9 @@ Unified Namespace.
 
 ## When should I use it?
 
-OPC UA is a communication protocol coming from the OT industry, so integration
-with IT tools is necessary to stream data from an OPC UA server. With Benthos
-UMH, you can easily connect to an OPC UA server, define the nodes you want to
-stream, and send the data to the Unified Namespace.
+Benthos UMH is valuable for integrating different protocols with the Unified Namespace.
+With it, you can connect to various protocol converters, define the data you want to
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md:25]: You cannot connect to various protocol converters, it is a protocol converter

File: umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md [hunk 3]
```diff
@@ -35,8 +33,8 @@ handling large amounts of data.
 
 Benthos UMH offers some benefits, including:
 
-- **Management Console integration**: Configure and deploy any number of
-  Benthos UMH instances directly from the Management Console.
+- **Management Console integration**: Configure and deploy any number of protocol converters via
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md:36]: should we add maybe the full list of what protocols one can read out? that is probably interesting for someone who is browsing through the documentation

File: umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md [hunk 4]
```diff
@@ -55,68 +53,65 @@ Benthos UMH offers some benefits, including:
...
 
-Mandatory fields are `opcuaID`, `enterprise`, `tagName` and `schema`. `opcuaID`
+Required fields are `opcuaID`, `enterprise`, `tagName` and `schema`. `opcuaID`
 is the NodeID in OPC-UA and can also be a folder (see [README](https://github.com/united-manufacturing-hub/benthos-umh?tab=readme-ov-file#node-ids)
 for more information). The remaining components are components of the resulting
 topic / ISA-95 structure (see also our [datamodel](/docs/datamodel)). By default,
-the schema will always be in _historian, and tagName is the keyname.
+the schema will always be in \_historian, and tagName is the keyname.
 
 ### Standalone
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md:93]: Do we need the standalone mode? With our universal protocol converter one can do everything protocol converter related than with the standalone version right?

File: umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md [hunk 5]
```diff
@@ -55,68 +53,65 @@ Benthos UMH offers some benefits, including:
...
+protocol converter:
 
-You first have to add a new connection to your OPC-UA server from the
-**Connection Management** tab.
+- **Universal Protocol Converter**: Opt for this choice if you need to configure
+  protocol converters for various supported protocols other than OPC-UA. This option
+  will prompt you to define the Benthos input and processor configuration in YAML format.
 
-![Add Connection](/images/features/data-connectivity-benthos/connection-management.png?width=80%)
+- **OPC-UA**: Select this option if you specifically need to configure
```
Comment (JeremyTheocharis) [inline umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md:68]: Should the user now use OPC-UA or the Universal Protocol converter? What is the difference?

(in OPC UA, there is for example a better data contextualization part that allows it easier to assign tags to certain places in the UNS)

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]

File: umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md (hunk 4 above)
Comment (ricci2511) [reply umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md:93]: Theoretically not, I left it in if for some reason a user has some very specific usecase where he prefers to deploy his own.

File: umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md (hunk 3 above)
Comment (ricci2511) [reply umh.docs.umh.app/content/en/docs/features/connectivity/benthos-umh.md:36]: Full might be rather large? We could list a few, and conclude with "and more" or something like that, as well as link to all available ones.

**Pull request merged**